### PR TITLE
#2356 Add SMB to backend.yml (docker-compose file)

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2356 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docker/backend.yml [hunk 1]
```diff
@@ -59,3 +59,20 @@ services:
       - POSTGRES_DB=${POSTGRES_DB:-ohara}
     ports:
       - 20000:5432
+
+  samba:
+    image: dperson/samba
```
Comment (chia7712) [inline docker/backend.yml]: no tag?

Comment (eechih) [reply docker/backend.yml]: 我已加入 Tag。（不過，該 Image 在 Docker hub 的 Tag 只有 latest，沒有其他 version 噢）

**Comment by vitojeng** [comment]
@chia7712 Does oharastream/ohara:deps and oharastream/ohara:ci need rebuild after merging this PR ?

**Comment by chia7712** [comment]
> Does oharastream/ohara:deps and oharastream/ohara:ci need rebuild after merging this PR ?

No. The rebuild is required only if the dependencies is changed.

**Pull request merged**